### ISSUE TITLE
Allow archive upload on image flash

### ISSUE THREAD

**Issue by aparcar** [openwrt/luci#5265]
Would it be possible to attach a config to the sysupgrade flash dialog? This way it wouldn't require a sysupgrade flash followed by a config upload but allow both to happen in a single step.

**Comment by pmelange** [comment]
I would also like to have this feature

**Comment by aparcar** [comment]
@jow- do you think this is easily possible?

**Comment by pmelange** [comment]
> does the sysupgrade script even support this?

Yes, it does. since commit https://github.com/openwrt/openwrt/commit/2b2884293af3706589dd586d04e4eafe9b98d0fc (13 years ago).
